pom.xml

```diff
@@ -327,53 +327,43 @@
           <excludedGroups>WatchPermissionAwareTest</excludedGroups>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <configuration>
+          <pom>
+            <includes>
+              <include>pom.xml</include>
+              <include>./**/pom.xml</include>
+            </includes>
+            <sortPom>
+              <expandEmptyElements>false</expandEmptyElements>
+            </sortPom>
+          </pom>
+          <java>
+            <googleJavaFormat>
+              <reflowLongStrings>true</reflowLongStrings>
+              <formatJavadoc>false</formatJavadoc>
```

Comment:
Why format javadoc is false?

Reply:
it was in the examples, but sure, why not